runtime-light/stdlib/string/regex-functions.h

```diff
@@ -50,8 +50,6 @@ inline bool valid_preg_replace_mixed(const mixed& param) noexcept {
 
 } // namespace regex_impl_
 
-using regexp = string;
```

Comment:
This should not be removed actually as `regexp` type is used in the compiler. Currently, it's turned off for `k2` mode, but we're gonna enable it in the future